src/backer/backering.py

```diff
@@ -296,6 +299,25 @@ def on_post(self, req, rep):
 
         cr = httping.parseCesrHttpRequest(req=req)
         serder = serdering.SerderKERI(sad=cr.payload, kind=eventing.Serials.json)
+
+        backer_identifier = serder.ked["b"]
```

Comment:
Ugh - `b` is an array. There should only be one but we need to (based on the other comment) get the backer identifier of this deployed cardano-backer, and search to make sure it's in `b` and set this as `backer_identifier`.

I don't feel like you are testing this properly if you were comparing this array with `item["bi"]`